spec/services/usps_in_person_proofing/proofer_spec.rb

```diff
@@ -161,6 +161,18 @@ def check_for_token_refresh_and_method_call(cache, redis)
         check_facility(facilities[0])
       end
 
+      it 'returns facilities sorted by ascending distance' do
+        stub_request_facilities_with_unorderd_distance
+        facilities = subject.request_facilities(location)
+
+        previous_post_office_distance = 0
+        facilities.count do |post_office|
+          current_distance = post_office.distance.delete_suffix(' mi').to_f
+          expect(previous_post_office_distance).to be <= current_distance
+          previous_post_office_distance = current_distance
+        end
```

Comment:
another way to test that they're sorted would be like an `each_cons`:

```suggestion
        expect(facilities.count).to be > 1
        facilities.each_cons do |facility_a, facility_b|
          expect(facility_a.distance).to be <= facility_b.distance
        end
```

Reply:
Less code and cleaner! I made this update too